federation/cluster/federation-up.sh

```diff
@@ -90,10 +90,9 @@ function init() {
       --dns-zone-name="${DNS_ZONE_NAME}" \
       --dns-provider="${DNS_PROVIDER}" \
       --image="${kube_registry}/hyperkube-amd64:${kube_version}" \
-      --apiserver-arg-overrides="--storage-backend=etcd2" \
       --apiserver-enable-basic-auth=true \
       --apiserver-enable-token-auth=true \
-      --apiserver-arg-overrides="--v=4" \
+      --apiserver-arg-overrides="--runtime-config=api/all=true,--v=4" \
```

Comment:
it was masked by the latter `--apiserver-arg-overrides="--v=4"` overrides (there were two) which hasn't been in effective since.